### DISCUSSION TITLE
Some sugestions

### DISCUSSION THREAD

**Discussion by LITUATUI** [razonyang/hugo-theme-bootstrap#611] [Ideas]
I recently found the Hugo theme [Geeky](https://github.com/statichunt/geeky-hugo) that is also based on bootstrap 5 and has some cool features that you might consider to adopt.

- Toggleable tabs with featured/recent posts with thumbnails on sidebar (looks great and saves space on the sidebar)
![geeky-tabs](https://user-images.githubusercontent.com/49299328/180268870-254a4d9c-e5d8-4a28-9b44-c5811d96142a.jpg)

- Related posts with thumbnails at the end of a post
![geeky-related-posts](https://user-images.githubusercontent.com/49299328/180269301-f14b9225-a3d1-48f3-afc0-06505d47ec51.jpg)

We usually see these features on WordPress blogs and news websites like [mine](https://pushevs.com/).

Your theme is already great, so no hurry :smiley:

**Comment by razonyang** [comment]
Looks good to me.

**Comment by razonyang** [comment]
Added the `posts toggle` widget via #612. ~~As for the `related posts` widget, currently I would keep it as it is.~~

The related posts style implemented by #880.
